azure-pipelines.yml

```diff
@@ -169,52 +187,14 @@ jobs:
   - bash: './determine-namespace.sh'
     name: DeployVars
     displayName: Set up environment variables for deployment
-  
```

Comment:
I don't think you should be removing all this?

Reply:
This was caused by merged changes into master after branch created. I have updated branch to reflect master